### PR TITLE
Using `GetAssembly` to get DLL loaction in ResNet18

### PR DESCRIPTION
**Pull request by LittleLittleCloud**

### CODE REVIEW

File: src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs [hunk 1]
```diff
@@ -25,7 +25,8 @@ public static class ResNet18Extension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> ResNet18(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return ResNet18(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "DnnImageModels"));
+            string fullPath = new DirectoryInfo(System.Reflection.Assembly.GetAssembly(typeof(DnnImageModelSelector)).Location).Parent.FullName;
```
Comment (eerhardt) [inline src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs]: Why are we just changing ResNet18, and not the other DnnImageFeaturizers?

File: src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs [hunk 2]
```diff
@@ -25,7 +25,8 @@ public static class ResNet18Extension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> ResNet18(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return ResNet18(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "DnnImageModels"));
```
Comment (eerhardt) [inline src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs]: We've gone through a few iterations of this code (see the git history on this file).

I don't see an issue being referenced for justifying this change. Can you add a scenario/reason why this is being changed?

Comment (LittleLittleCloud) [reply src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs]: Sure
#4173

File: src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs (hunk 1 above)
Comment (LittleLittleCloud) [reply src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs]: Just lazy.... I will change the other two after you think the change is necessary...

**Pull request closed without merging**